netbox/core/tables/plugins.py

```diff
@@ -86,3 +87,10 @@ class Meta(BaseTable.Meta):
         # List installed plugins first, then certified plugins, then
         # everything else (with each tranche ordered alphabetically)
         order_by = ('-is_installed', '-is_certified', 'name')
+
+    def render_title_long(self, value, record):
+        if record.disabled:
+            return f"{value}"
+
+        url = reverse('core:plugin', args=[record.config_name])
+        return mark_safe(f"<a href='{url}'>{value}</a>")
```

Comment:
I think we can obviate the need for a custom method by passing a lambda to `linkify` on the column:

```python
linkify=lambda record: not record.disabled
```

Reply:
I'm not sure this will work, it can't use get_absolute_url as it's not a model so the url accessor has to be provided (as below) - I don't think it can be overloaded with this and the test for record.disabled?

```
    title_long = tables.Column(
        linkify=('core:plugin', [tables.A('config_name')]),
        verbose_name=_('Name')
    )
```

Reply:
Ah right, thought we had a URL method on the plugin record.